package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "squad-tests",
```

Comment:
What's your npm version? I noticed "name" and "version" (and other parameters) are not included in package-lock.json. When I ran `npm install` I got lots of changes on _package-lock.json_. These are mine:
```
$ npm -v
6.4.1
$ node -v
v10.12.0
```

Reply:
Same versions here. Can you send the diff of the package-lock file? Maybe some pkgs got updated in the meantime...

Reply:
Here's the diff: https://gist.github.com/chaws/3a3f3266b8834eaa5a909866f7fab65b

Reply:
Updated the -lock file.

Reply:
worked alright now!